tests/test_demod_mapmaker.py

```diff
@@ -45,7 +45,7 @@ def make_map(obs, nside=None, nside_tile=None, shape=None, wcs=None, comps='TQU'
 
     signals    = [signal_map]
     mapmaker   = DemodMapmaker(signals, noise_model=NmatWhite(), comps=comps)
-    mapmaker.add_obs('obs0', obs, use_psd=False)
+    mapmaker.add_obs('obs0', obs, use_psd=False, apply_wobble=False)
```

Comment:
You've modified this test in order to suppress wobble correction. 

Do you have a test that _does_ do wobble correction?  Seems like that would be good to have.